erts/emulator/beam/jit/x86/beam_asm_perf.cpp

```diff
@@ -137,7 +137,6 @@ class JitPerfDump {
         for (BeamAssembler::AsmRange &r : ranges) {
             size_t nameLen = r.name.size();
             ptrdiff_t codeSize = (char *)r.stop - (char *)r.start;
-            ASSERT(codeSize > 0);
```

Comment:
This assert was added because perf does not like it when the codeSize is a negative number and we had mistakes where it was. Have you found any cases where this assert triggers for negative numbers, or only for when codeSize == 0?

I think it would be better to just change it to `codeSize >= 0`.

Reply:
I've only seen zero. Changing to "codeSize >=0" works for me too, I'll update the PR.